lib/galaxy/config/__init__.py

```diff
@@ -1057,11 +1056,6 @@ def _process_config(self, kwargs):
         # Statistics and profiling with statsd
         self.statsd_host = kwargs.get("statsd_host", "")
 
-        ie_dirs = self.interactive_environment_plugins_directory
-        self.gie_dirs = [d.strip() for d in (ie_dirs.split(",") if ie_dirs else [])]
```

Comment:
`gie_dir` seems to be used in `lib/galaxy/webapps/galaxy/controllers/visualization.py`